core/src/main/proto/wallet.proto

```diff
@@ -512,3 +512,10 @@ message ExchangeRate {
 
   // Next tag: 4
 }
+/** CoinJoin KeyChain */
+message CoinJoin {
+  // key chain
+  repeated Key key = 1;
+  // last value of rounds from mixing
+  required int32 rounds = 2;
+}
```

Comment:
`rounds` is used to calculate the balance of coinjoin coins.

if all coins were mixed for 4 rounds, but the `rounds = 5` then the coinjoin balance will be zero.  This value needs to persist.